dev/manual_tests/lib/actions.dart

```diff
@@ -355,18 +353,14 @@ class FocusDemo extends StatefulWidget {
 }
 
 class _FocusDemoState extends State<FocusDemo> {
-  FocusNode outlineFocus;
-  UndoableActionDispatcher dispatcher;
-  bool canUndo;
-  bool canRedo;
+  final FocusNode outlineFocus = FocusNode(debugLabel: 'Demo Focus Node');
+  late final UndoableActionDispatcher dispatcher = UndoableActionDispatcher();
+  late bool canUndo = dispatcher.canUndo;
+  late bool canRedo = dispatcher.canRedo;
 
   @override
   void initState() {
     super.initState();
-    outlineFocus = FocusNode(debugLabel: 'Demo Focus Node');
-    dispatcher = UndoableActionDispatcher();
-    canUndo = dispatcher.canUndo;
-    canRedo = dispatcher.canRedo;
```

Comment:
Actually I think you should leave the `initState` as it was and initialize as nonnull to false.
```dart
  bool canUndo = false;
  bool canRedo = false;
```